src/blocks/blocks/stripe-checkout/inspector.js

```diff
@@ -13,14 +13,77 @@ import {
 	SelectControl,
 	Spinner,
 	TextControl,
-	TextareaControl
+	TextareaControl,
+	ExternalLink
 } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
-import { ButtonToggleControl, RichTextEditor } from '../../components/index.js';
-import { useState } from '@wordpress/element';
+import { ButtonToggleControl, Notice as OtterNotice, Notice, RichTextEditor } from '../../components/index.js';
+import { Fragment, useContext, useState } from '@wordpress/element';
```

Comment:
NIT, we can move WP dependencies out of internal dependencies, so it's easy to notice if they're already imported or need importing.